tests/e2e/util.go

```diff
@@ -3358,7 +3358,7 @@ func hostLogin(user, instruction string, questions []string, echos []bool) (answ
 
 // runCommandOnESX executes ssh commands on the give ESX host and returns the bash
 // result.
-func runCommandOnESX(username string, addr string, cmd string) (string, error) {
+func runCommandOnESX(username string, addr string, cmd string, ignoreExitCode ...bool) (string, error) {
```

Comment:
when I look at it again now, I see that we are not only ignoring status code here, we are also ignoring all the errors with this flag. Add a TODO here to fix it properly and raise a bug to address it for now. Also, for now lets rename the variable to indicate what happens when we use it.